src/components/Dropdown/props.ts

```diff
@@ -230,5 +230,10 @@ export type DropdownProps = {
     * The default behavior is to create a div element and append it at the end of the body,
     *  but you can reset it to the scrolling area and make a relative reposition.
     */
-   getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement
+   getPopupContainer?: (triggerNode: HTMLElement) => HTMLElement,
+
+    /**
+      * The placement of the dropdown menu, can be: `topLeft`, `topRight`, `bottomLeft`, `bottomRight`, `top`, `bottom`.
+      */
+   placement?: 'topLeft' | 'topRight' | 'bottomLeft' | 'bottomRight' | 'top' | 'bottom'
```

Comment:
please define an enum and export it, so that the prop can be used with something like:

```
placement={Dropdown.Placement.TopLevel
```

See Triggers for examples

Reply:
Done. I also moved all the typings to a separate file, and renamed the props file to be consistent with the other DS components

Reply:
> > and renamed the props file to be consistent with the other DS components
> 
> this was not to be done as we decided to shift away from that pattern

undone